examples/k8eraid-deployment.yml

```diff
@@ -26,7 +26,7 @@ spec:
       containers:
         - name: k8eraid
           command: ['/k8eraid']
-          image: artifactory.inf.bloomberg.com/kubernetesinfrastructure/k8eraid:latest
+          image: bloomberg/k8eraid:0.8.1
```

Comment:
Should this be 'latest' or no tag at all?

Reply:
Actually, it's best to use tagged versions in this case. `latest`/no tag is bad practice in k8s world.

_However_, the tag is wrong. Fixing.